directord/__init__.py

```diff
@@ -430,6 +431,38 @@ def purge_jobs(self):
             "success"
         ]
 
+    def bootstrap(self, catalogs=None):
+        """Run cluster wide bootstrap operations.
+
+        Runs bootstrap operations from the directord library and returns a
+        Tuple of all bootstrapped nodes. This method requires a list of
+        catalogs entries which are expected to be fully qualified files.
+
+        >>> lib.bootstrap(catalogs=["/path/file1"])
+
+        If the catalogs parameter is None, this method will raise a Syntax
+        Error.
+
+        :param catalogs: List of files parsed as bootstrap catalogs
+        :type catalogs: List
+        :returns: Tuple
+        """
+
+        if not catalogs:
+            raise SyntaxError("Missing catalog data.")
+
+        self.args.catalog = list()
+        try:
+            for c in catalogs:
+                self.args.catalog.append(open(c))
```

Comment:
Doesn't the open() need encoding to make pylint happy now?

Reply:
Not that I'm aware of?